DrawerKitDemo/DrawerKitDemo/PresentedViewController.swift

```diff
@@ -22,6 +22,10 @@ class PresentedViewController: UIViewController {
             }
         }
     }
+
+    @IBAction func unwindFromModal(with segue: UIStoryboardSegue) {
+        print("Unwound from the modal.")
```

Comment:
Is this going to be removed at some point?

Reply:
No. It is the Storyboard unwind point for dismissing the modal in the demo.

Reply:
I meant the print statement.